subprojects/parseq-restli-client/src/main/java/com/linkedin/restli/client/ParSeqRestClient.java

```diff
@@ -79,6 +94,7 @@ public class ParSeqRestClient extends BatchingStrategy<RequestGroup, RestRequest
     _requestConfigProvider = requestConfigProvider;
     _requestContextProvider = requestContextProvider;
     _d2RequestTimeoutEnabled = d2RequestTimeoutEnabled;
+    _executor = DirectExecutor.getInstance();
```

Comment:
can we call the new constructor instead of duplicate assignments here

Reply:
Sure. Updated the PR.